packages/devtools_app/lib/src/version.dart

```diff
@@ -101,6 +101,22 @@ class SemanticVersion implements Comparable {
   bool isSupported({@required SemanticVersion supportedVersion}) =>
       compareTo(supportedVersion) >= 0;
 
+  bool operator <(other) {
```

Comment:
there should be some mixin that derives all of these from compareTo for. If not, you should add one.

Reply:
added mixin